package.json

```diff
@@ -24,12 +24,15 @@
     "html-webpack-plugin": "^3.2.0",
     "mocha": "^5.2.0",
     "mochapack": "^1.0.0",
+    "moment": "^2.28.0",
     "node-sass": "^4.14.1",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.1",
     "webpack": "^4.31.0",
     "webpack-cli": "^3.3.11",
     "webpack-dev-server": "^3.11.0"
   },
-  "dependencies": {}
+  "dependencies": {
```

Comment:
Do we need this in dev-dependencies _and_ dependencies?

Reply:
Honestly? I am not sure. I will create this as an issue and tag it with `question` to look into it further.